### ISSUE TITLE
[Refactor] Clean up JSX in container files

### ISSUE THREAD

**Issue by codytowstik** [taniarascia/takenote#227]
For example `AppSidebar.tsx` and `LandingPage.tsx`

**Problem**
It's hard to follow some of the logic in ~200 lines of JSX

**Suggested Approach**
- Find logically or semantically related elements and move them into a separate component
For example, maybe move the elements with class `app-sidebar-link` in `AppSidebar.tsx` into a new component

**Note**
This should probably be done on a file by file basis, so if you are refactoring a file please comment on this issue.

*If there aren't unit tests* for the part you change, please add them.

**Comment by taniarascia** [comment]
This is a good idea. The main app files were built out quickly and not separated into logical, reusable components for the most part.
